### DISCUSSION TITLE
Student Developer Pack stuck on processing information

### DISCUSSION THREAD

**Discussion by robinhickmann** [community/community#102532] [GitHub Education]
### Hi everyone,

I was trying to apply for the Student Developer Pack and when I got the the upload proof section I uploaded my school id card, selected "Dated school ID" and then clicked "Process my application". At the bottom of the page, the description above the button says "Please wait as processing can take several seconds." I have waited for over 20 minutes and the button still says "Processing information...". I have also tried reloading the page and uploading the image again, even trying a different file format. I also tried checking the status which I saw a link for on some github support site. When I go to that link it leads me to my current application but without the photo and proof type entered. I tried once again uploading the image and selecting the proof type, still nothing. For some context, I'm from sweden and when I typed in my school it didn't come up on the map but when I tried specifing an address it didn't let me proceed until I selected one of the incorrect schools. I did however provide as much info about my school as possible.

**Comment by naveen-98** [comment]
Hello @SandwichBtw !

It's definitely frustrating to encounter issues with your Student Developer Pack application, especially after taking the initiative to try different solutions. Here are some additional steps you can take to try and resolve the processing delay:

1. Contact GitHub Support:

Directly reach out to GitHub Support through their contact form: https://docs.github.com/support/contacting-github-support
Explain your situation in detail, including the specific proof you uploaded, the school issue you faced, and the current "Processing information..." status.
Provide screenshots or recordings if possible to illustrate the problem better.
2. Try alternative verification methods:

If you have a school-issued email address, consider using that for verification instead of uploading your ID.
If you are part of a student organization or club, see if they can provide you with official documentation proving your status.
3. Check for updates on the application status:

Regularly check the link you mentioned that leads to your application.
Sometimes, it might take a while for the verification process to complete, and the status might update later.
4. Consider contacting your school:

Explain the situation to your school's IT department or administrative office.
They might be able to provide you with an official document or letter confirming your enrollment, which you can then upload as proof.
Additional tips:

Ensure the uploaded photo is a clear and legible image of your ID card.
Make sure the file format is recognized by GitHub, like JPG, PNG, or PDF.
Check for any temporary outages or errors reported on GitHub's status page: https://www.githubstatus.com/
Remember, it's important to remain patient and polite while communicating with GitHub Support or your school. Persistence and providing clear information can help them understand and resolve your issue as quickly as possible.

Good luck!
